build-system/common/update-design-review-issues.js

```diff
@@ -31,18 +31,25 @@
  */
 const https = require('https');
 
-const dayOfWeek = /* wednesday */ 3; // sunday = 0, monday = 1, ...
+/** @typedef {0|1|2|3|4|5|6} */
+let DayOfWeekDef; // sunday = 0, monday = 1, ...
 
 const sessionDurationHours = 1;
 
-// Times in this rotation are adjusted according to Daylight Savings
+/** @typedef {[DayOfWeekDef, string, string]} */
+let RotationItemDef;
+
+/**
+ * Times in this rotation are adjusted according to Daylight Savings
+ * @type {Array<RotationItemDef>}
+ */
 const timeRotationUtc = [
```

Comment:
The result of this rotation remains unchanged.
The code change is only to sort the list so that times are incremental, and adjusting the `start` pivot to match.